src/utilities.jl

```diff
@@ -45,6 +55,14 @@ Equivalent to `permutedims(x, perm)`.
 """
 swapdims(perm) = Base.Fix2(permutedims, perm)
 
+"""
+    applyactivation(activation, x)
+
+Apply an activation function to a given input.
+Equivalent to `activation.(x)`.
+"""
+applyactivation(activation, x) = activation.(x)
```

Comment:
Is this used somewhere? Did I miss it? I also have trouble seeing how this improves on just writing the broadcast. I would either just do `activation.(x)` or `x -> activation.(x)` in the two scenarios I can think up. Both options are shorter than this.

Reply:
This makes the error message (in case there is one) more readable, since with the anonymous function all you can see is `var#XXX`

Reply:
Wouldn't `x -> applyactivation(activation, x)` have the same issue?

Reply:
Right now it's `Base.Fix1`, which I'm _hoping_ will error slightly more clearly. But see https://github.com/FluxML/Metalhead.jl/pull/170#discussion_r900694370

Reply:
The easiest is to just make the default activations broadcast on arrays (which I think NNlib already does for some of them). Better might be adding error hinting to them (i.e. `relu6(::AbstractArray)` -> "MethodError: maybe you intended to broadcast relu6 over an array?"

Reply:
Hmm, true. This might need to be taken up in NNlib then. I'll do away with `applyactivation` for now and make it a plain broadcast

Reply:
https://github.com/FluxML/NNlib.jl/issues/422